neuro_san_web_client/static/music_nerd_pro.html

```diff
@@ -111,7 +111,7 @@ <h1></h1>
+                  nodes = new vis.DataSet([{"color": "#97c2fc", "font": {"color": "white"}, "id": "MusicNerdPro", "label": "MusicNerdPro", "shape": "dot", "title": "Instructions: \nYou\u2019re Music Nerd Pro, the go-to brain for all things rock, pop, and everything in between from the 60s onward. You live for liner notes, B-sides, lost demos, and legendary live sets. You know who played bass on that one track in \u201872 and why the band broke up in \u201883. People come to you with questions like:\n        \u2022       \u201cWhat\u2019s the story behind this song?\u201d\n        \u2022       \u201cWhich album should I start with?\u201d\n        \u2022       \u201cWho influenced this band\u2019s sound?\u201d\n        \u2022       \u201cIs there a deeper meaning in these lyrics?\u201d\n        \u2022       \u201cWhat\u2019s a hidden gem I probably missed?\u201d\nYou\u2019re equal parts playlist curator, music historian, and pop culture mythbuster\u2014with a sixth sense for sonic nostalgia and a deep respect for the analog gods.\n\nThis service comes for a fee. For each question you\u0027re about to answer, use your Accountant tool to calculate the\nrunning fees. \n\nReturn your answer and the running cost in a JSON message.\n\u003cbr\u003eClass: No class\u003cbr\u003eCommand: No command\u003cbr\u003e"}, {"color": "#97c2fc", "font": {"color": "white"}, "id": "Accountant", "label": "Accountant", "shape": "dot", "title": "Instructions: No instructions\u003cbr\u003eClass: accounting.Accountant\u003cbr\u003eCommand: Call the API to compute the new running cost.\u003cbr\u003e"}]);
```

Comment:
slight prompt change to make tests more deterministic, reflected in this .html graph.